File: exercicios/1046/1046.py
x = input().split()

a = int(x[0])
b = int(x[1])

dia = 24;

if((a >= b)):
    x1 = dia - ( a - b)
    print("O JOGO DUROU", x1," HORA(S)")

elif(a <= b):
    x1 = (b - a)
    print("O JOGO DUROU", x1,"HORA(S)")
elif(a == b) <= dia:
    print("O JOGO DUROU 24 HORA(S)")

File: exercicios/JavaScript/1046/1046.js
var input = require('fs').readFileSync('stdin','utf8');
var lines = input.split('\n');

var line1 = lines.shift().split(' ');

var a = parseInt(line1[0])
var b = parseInt(line1[1])

if(a == b){
    console.log("O JOGO DUROU 24 HORA(S)");
}else if(a < b){
    var tempo = parseInt(b - a);

    console.log("O JOGO DUROU " + tempo + " HORA(S)");
}else if(a > b){
    var tempo = parseInt(24 - (a - b));

    console.log("O JOGO DUROU " + tempo + " HORA(S)");
}